src/dlls/mscoree/mscoree.cpp

```diff
@@ -181,21 +175,12 @@ BOOL WINAPI DllMain(HANDLE hInstance, DWORD dwReason, LPVOID lpReserved)
             {
                 return FALSE;
             }
-
-#ifdef FEATURE_MERGE_JIT_AND_ENGINE
```

Comment:
I don't think this will break the closed build: if it was not calling jitStartup, it would already be broken (the JIT would assert in the allocator the first time compileMethod was called).